README.md

```diff
@@ -1,56 +1,31 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
 
+# Hello Microverse
 > Description the project.
```

Comment:
- [ ] Your README is documented well💯, but I think it would be better to add a description to your project 👌